test/integration/telemetry/pages/warning.skip

```diff
@@ -1,7 +1,7 @@
-function a(v) {
-  return v
-}
 ;['index.js'].forEach(async f => {
+  function a(v) {
+    return v
+  }
   await import(a('./dynamic-file-imports/' + f))
```

Comment:
Is this change needed? If not, better revert it

Reply:
It's required because after tree shaking turbopack cannot evaluate `a('./dynamic-file-imports/' + f)` due to `a` being moved to another module.